### DISCUSSION TITLE
Versions

### DISCUSSION THREAD

**Discussion by ostap667inbox** [civitai/civitai#420] [General]
Sorry guys, but this new interface is disgusting in terms of UI/UX.
You turned a normal looking list that was easy to use into a horrible carousel with arrows.
This is not the kind of content you want to squeeze into a tiny screen area at the expense of functionality. Being able to see the version list is IMPORTANT.
![civitai-versions](https://user-images.githubusercontent.com/123593542/229897882-605c2f6d-b7a6-4fd6-9e14-199cb1d085c1.png)

**Comment by TroubleDarkness** [comment]
Yeah, the worst redisign you can imagine. All UI elements are squeezed now, so you need to open every element on mix/lora/etc pages. Like reviews, version infos. Gallery are now separated for different versions and still have a description with version name. The only thing that seems better to me - editing.
